docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md

```diff
@@ -0,0 +1,82 @@
+# Install the Open Enclave SDK (Ubuntu 20.04)
+
+## Platform requirements
+
+- Ubuntu 20.04-LTS 64-bit.
+- SGX1-capable system with support for Flexible Launch Control (FLC).
+    - You can acquire a VM with the required features from [Azure Confidential Compute](https://azure.microsoft.com/en-us/solutions/confidential-compute/).
+    - If you are setting up your own device, [check if your existing device supports SGX with FLC](/docs/GettingStartedDocs/Contributors/building_oe_sdk.md#1-determine-the-sgx-support-level-on-your-developmenttarget-system).
+        - If your device only supports SGX without FLC, you will need to [clone and build OE SDK](/docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md) for that configuration.
+        - If your device does not support SGX, follow the [instructions for simulation mode](/docs/GettingStartedDocs/install_oe_sdk-Simulation.md). Please see [instructions for determining SGX support](/docs/GettingStartedDocs/SGXSupportLevel.md) on the machine you are using.
+
+### 1. Configure the Intel and Microsoft APT Repositories
+```bash
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+
+echo "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-10 main" | sudo tee /etc/apt/sources.list.d/llvm-toolchain-focal-10.list
+wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+
+echo "deb [arch=amd64] https://packages.microsoft.com/ubuntu/20.04/prod focal main" | sudo tee /etc/apt/sources.list.d/msprod.list
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+
+sudo apt update
+```
+
+### 2. Install the Intel SGX DCAP Driver
+Some versions of Ubuntu come with the SGX driver already installed. You can check
+by running with the following:
+
+```
+$ dmesg | grep -i sgx
+[  106.775199] sgx: intel_sgx: Intel SGX DCAP Driver {version}
+```
+
+If the output of the above is blank, you should proceed with installing the driver:
+
+```bash
+sudo apt update
+sudo apt -y install dkms
+wget https://download.01.org/intel-sgx/sgx-dcap/1.9/linux/distro/ubuntu20.04-server/sgx_linux_x64_driver_1.36.2.bin -O sgx_linux_x64_driver.bin
+chmod +x sgx_linux_x64_driver.bin
+sudo ./sgx_linux_x64_driver.bin
+```
+
+> This may not be the latest Intel SGX DCAP driver.
+> Please check with [Intel's SGX site](https://01.org/intel-software-guard-extensions/downloads)
+> if a more recent SGX DCAP driver exists.
+
+### 3. Install the Intel and Open Enclave packages and dependencies
+```bash
+sudo apt -y install clang-8 libssl-dev gdb libsgx-enclave-common libsgx-quote-ex libprotobuf10 libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client open-enclave
```

Comment:
To change to clang-10 when https://github.com/openenclave/openenclave/pull/4051 is merged